### PR TITLE
Eloquent js chapter3

### PR DESCRIPTION
**Pull request by DGEs2018**
Could you please re-review these files with the latest fixed branch name `eloquent-js-chapter3` instead of `eloquent-js-chapter4` (the link to the older one - https://github.com/DGEs2018/Traineeship-progress/pull/9). I appreciate it.

### CODE REVIEW

File: Week8/bean-counter.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/*
+Write a function that takes in two string parameters, with the second one containing one of the
+string characters to be counted. 
+*/
+function countChar(string, character) {
+	let charQuantity = 0;
+	for (let i = 0; i < string.length; i++) {
+		if (string[i] == character) {
```
Comment (sophialittlejohn) [inline Week8/bean-counter.js]: One more place where I'd recommend using `===`

Comment (DGEs2018) [reply Week8/bean-counter.js]: I just fixed that. Thank you !!

File: Week8/journal.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# 21.05.2019
+* Eloquent JavaScript Chapter - 4 Data structiures: objects and arrays
```
Comment (aiwatko) [inline Week8/journal.md]: So seems the pull request is related to chapter 3, whereas your journal points to chapter 4. where can I find your notes from chapter 3? :)

**Comment by aiwatko** [comment]
Good job!

**Pull request merged**